scripts/setup_influxdb.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+. "$(dirname "$0")"/_select_docker_compose.sh
+. "$(dirname "$0")"/_load_dot_env.sh $1
+
+echo "Using docker compose file at: ${2}"
+
+# Add influxdb users with write access for optimizer/simulator and with admin rights for the frontend (root admin user via env vars)
+$DOCKER_COMPOSE -f ${2} up -d --wait omotes_influxdb
```

Comment:
You can remove `-d`, it is implied by `--wait`.

Reply:
Fixed by https://github.com/Project-OMOTES/computation-engine/pull/43/commits/52b8678aacc51b811e013eb0e6cbe4ee088f81f0